### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ New Project_ Error Identification: There is no visual indication  provided for the mandatory fields(Group Id, Artifact Id).

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3354]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS3.3.1; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated**: 3.3.1

**Repro Steps:**
1) Open IntelliJ IDEA application.
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup.
3) Tab till 'Plugins' tab and press enter.
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in.
7) Sign in with any Azure subscription.
8) Tab till File menu and click new-->Project.
9) In the New Project dialog box, select Maven and press enter.
10) Tab till maven-archetype-web app and tab till Next and press enter.
11)Verify any visual indication is provided for mandatory fields(Group Id, Artifact Id).

**Actual Result:**
There is no visual indication  provided for the mandatory fields(Group Id, Artifact Id).

**Expected Result:**
There should be a visual indication to represent mandatory fields. User should know the mandatory fields when the keyboard focus is on those controls not at the time of going to nest screen.

**User Impact:** 
User should know about mandatory fields when focus is on those controls so that they will fill required details, otherwise they again need to all controls to give the info into mandatory fields if the error comes at the time of navigating to the next field.

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={733120d0-56c5-479c-889f-fee36ea08ee0}
![Bug_3354_Error Identification](https://user-images.githubusercontent.com/38523250/62207419-adebb600-b3b1-11e9-8d47-bf85befb0f1b.png)

**Comment by andxu** [comment]
Hi @Accessibility56, this ui is provided by idea other than our product, my fresh installation of idea can reproduce the same issue.
![image](https://user-images.githubusercontent.com/17075863/64595278-319eb680-d3e4-11e9-921c-b859b0966a6c.png)

**Comment by andxu** [comment]
* Importance of experience: Little, in other tutorials and other tutorials targeting create maven project  with archetype in idea will teach users to specify groupId/artifactId/version(which are very important in maven), on the other hand, user will got the proper error message when he presses enter.  
* Quality of experience: Little, users will got a proper error message when user presses enter. 
 Alternate solutions: No alternate technologies or options can be leveraged.

**Comment by Accessibility56** [comment]
@andxu Please provide the link to AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Accessibility56** [comment]
External bug is logged https://youtrack.jetbrains.com/issue/IDEA-222348

**Comment by Flanker32** [comment]
@Accessibility56 This issue is linked to AAT, please help us review it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749788
